packages/state-management/src/action-creators.ts

```diff
@@ -130,6 +130,8 @@ export function updateSiteInfo(): ThunkAction<void, State, unknown, AnyAction> {
                         stripeKey,
                         codeInjectionHead,
                         codeInjectionBody,
+                        mailingAddress,
```

Comment:
Why did we need `mailingAddress` here?

Reply:
We also need to update `mailingAddress` if some changes happen in mailingAddress. I just saw that the mailingAddress is missing here so I added it.

Reply:
Great. Thanks.